iot-hub/Samples/service/ImportExportDevicesWithManagedIdentitySample/Parameters.cs

```diff
@@ -28,16 +28,10 @@ internal class Parameters
             HelpText = "The Uri of storage container for import and export jobs.")]
         public string BlobContainerUri { get; set; }
 
-        [Option(
```

Comment:
We're removing the option to run this sample without any kind of identity?

Reply:
This was something I added earlier. This is not something the API requires. It just makes the sample harder to understand. The sample explicitly says this is for managed identites only. So the StorageAuthenticationType is set to IdentityBased in JobProperties.

Reply:
The API does not ask if you want to do system-defined or user-defined. That is dictated by the fact that Identity property is present or not on the request body. I have documented that in detail.